mlflow/pyfunc/cli.py

```diff
@@ -40,14 +90,23 @@ def serve(model_path, run_id, port):
               required=True)
 @click.option("--output-path", "-o", help="File to output results to as CSV file." +
                                           " If not provided, output to stdout.")
-def predict(model_path, run_id, input_path, output_path):
```

Comment:
It would be nice to add no-conda to mlflow/utils/cli_args.py and reference it here although we can do that later.